docs/sources/datasources/prometheus/configure-prometheus-data-source.md

```diff
@@ -63,15 +63,11 @@ The first option to configure is the name of your connection:
 
 - **Default** - Toggle to select as the default name in dashboard panels. When you go to a dashboard panel this will be the default selected data source.
 
-### HTTP section
+### Connection section
 
-- **URL** - The URL of your Prometheus server. If your Prometheus server is local, use `<http://localhost:9090>`. If it is on a server within a network, this is the URL with port where you are running Prometheus. Example: `<http://prometheus.example.orgname:9090>`.
+- **Prometheus server URL** - The URL of your Prometheus server. If your Prometheus server is local, use `<http://localhost:9090>`. If it is on a server within a network, this is the URL with port where you are running Prometheus. Example: `<http://prometheus.example.orgname:9090>`.
```

Comment:
I've had some difficulty with this documentation, where I've run Grafana and Prometheus as different Docker containers on the same host, and the connectivity was not working. [This was exactly the issue I was having.](https://github.com/grafana/grafana/issues/46434)

So I was wondering if it would be a good idea to add a note to this documentation to direct people to `http://host.docker.internal:9090` if they are experiencing the [above issue]((https://github.com/grafana/grafana/issues/46434)).

It seems like a lot of people have had difficulties with this, and I think it would be a better experience for users to include it in the official documentation.

What do you think? 😃

Reply:
I think that information could be useful, yeah. Something like:

```
{{< admonition type="note" >}}
If you're running Grafana and Prometheus in containers, the localhost of one container is not the localhost of another container, even if you published the port to the host – you can't reach the Prometheus container or the host using localhost from the Grafana container.

You need to use the IP address of the Prometheus container, or the hostname if you are using Docker Compose.
{{< /admonition >}}
```